app/controllers/advertisements_controller.rb

```diff
@@ -0,0 +1,28 @@
+class AdvertisementsController < ApplicationController
+  def index
+    @advertisements = Advertisement.all
+  end
+
+  def show
+    @advertisement = Advertisement.find(params[:id])
+  end
+
+  def new
+    @advertisement = Advertisement.new
+  end
+
+  def create
+    @advertisement = Advertisement.new
+    @advertisement.title = params[:advertisement][:title]
+    @advertisement.copy = params[:advertisement][:copy]
+    @advertisement.price = params[:advertisement][:price]
```

Comment:
Lines 16 - 18 look a bit repetitive. There's a shortcut you can use here.  `Advertisement.new` can optionally take a hash of parameters.  `params[:advertisement]`  is also conveniently a hash.  

See if you can find a shortcut and we'll look at it together on Friday.